content/references/data-api.md

```diff
@@ -2736,86 +2763,37 @@ A successful response uses the HTTP code **200 OK** and has a JSON body with the
 | Field                   | Value                           | Description      |
 |:------------------------|:--------------------------------|:-----------------|
 | `result`                | String                          | The value `success` indicates that the body represents a successful response. |
-| `last_datetime`         | Integer                         | Number of links returned. |
-| `validation_public_key` | String - Base-58 [Public Key][] | Validator public key of this validator. |
-
-
-#### Example
+| `count`                 | Integer                         | Number of validators returned. |
+| `validators`            | Array of [Validator Objects][]  | List of validators active in the last 24 hours. |
```

Comment:
`[Validator Objects][]` isn't showing up as a link in the generated HTML. Do we want to change this to [Validator Objects](#validator-objects)? Or handle it some other way?